docs/DeveloperGuide.adoc

```diff
@@ -77,6 +77,37 @@ Use case ends.
 +
 Use case resumes at step 2.
 
+
+=== Use case: Renaming an existing tag
+
+*MSS*
+
+. User requests to see list of tags
+. AddressBook shows a list of tags
+. User requests to rename a specific tag
+. AddressBook requests user to enter a new name
+. User enters the new name of tag
+. AddressBook requests for confirmation of the change
```

Comment:
If you have this step, the next one should be something like `User confirms the change`.
Remember, use cases depict the interaction between the user and the system